app/views/organisations/_show_organisation.html.erb

```diff
@@ -16,6 +16,10 @@
   <%= render partial: 'what_we_do' %>
 <% end %>
 
+<% if @organisation.is_promotional_org? %>
+  <%= render partial: 'promotional_features' %>
```

Comment:
This means this will render even if the promotional org does not have any promotional features - is it worth doing a check here or in the promotional_features partial?